ldi-orchestrator/ldio-connectors/ldio-ldes-client/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioLdesClientController.java

```diff
@@ -0,0 +1,44 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.requestexecutor.executor.RequestExecutor;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.services.ComponentExecutor;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.types.LdioInput;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.valueobjects.ComponentProperties;
+import ldes.client.treenodesupplier.domain.valueobject.StatePersistence;
+
+import java.util.concurrent.ExecutorService;
+
+import static java.util.concurrent.Executors.newSingleThreadExecutor;
+
+public class LdioLdesClientController {
```

Comment:
I find this name very confusing.. 
I would have thought this is the REST endpoints controller of the connector, but this is the class that executes the LDES Client?